### ISSUE TITLE
Give colored cosmetics with commands

### ISSUE THREAD

**Issue by Agaloth** [PluginBugs/Issues-CosmeticsCore#166]
### Terms

- [X] I'm using the very latest version of CosmeticsCore and its dependencies.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-CosmeticsCore/issues) to check if the same suggestion was already reported.
- [X] I already searched on the [plugin wiki](https://lonedev.gitbook.io/cosmeticscore/) to know if the feature is already implemented.
- [X] I already searched on the **Discord** server to know if anyone already has a solution for this or if the feature was already suggested.

### Discord tag (optional)

Agaloth

### Describe the solution you'd like

Basically, right now you can give cosmetics but you're not able to specify the RGB color you want with the command, this is useful for bedrock players as the wardrobes won't work on bedrock edition so that system could be great for a GUI-based approach with an NPC.

### Is your feature request related to a problem?

Yes, mainly with bedrock players.

### Describe alternatives you've considered

Using a GUI that gives the cosmetics with the give command in cosmeticscore rather than /wardrobe

**Comment by 0000Artur0000** [comment]
It would be cool if there was a command to issue like this: 
/cosmetics <player> add <name> [silent|not-silent] (#FF0000)

**Comment by 0000Artur0000** [comment]
Please
